internal/notifier/gitea.go

```diff
@@ -0,0 +1,176 @@
+package notifier
+
+import (
```

Comment:
Please arrange the imports here the same way as in the other files of this package. https://github.com/golang/go/wiki/CodeReviewComments#imports